src/Connector.php

```diff
@@ -36,6 +36,7 @@ public function __construct(LoopInterface $loop, array $options = array())
 
             'dns' => true,
             'timeout' => true,
+            'happy_eyeballs' => \PHP_VERSION_ID < 70000 ? false : true,
```

Comment:
You can drop the ternary operator.
```suggestion
            'happy_eyeballs' => \PHP_VERSION_ID >= 70000,
```